modules/ecommerce/pmpro/class-swsales-module-pmpro.php

```diff
@@ -495,6 +497,27 @@ public static function admin_enqueue_scripts() {
 		}
 	} // end admin_enqueue_scripts()
 
+	/**
+	 * Whether the current sitewide sale should be hidden.
+	 * Callback for the swsales_hide filter.
+	 */
+	public static function swsales_hide( $hide_sale, $sitewide_sale ) {
+		// Get the meta value for levels this sale should be hidden for.
```

Comment:
```suggestion
		if ( 'pmpro' !== $sitewide_sale->get_sale_type() ) {
 			return $show_banner;
 		}

		// Get the meta value for levels this sale should be hidden for.
```
Make sure that we have a PMPro sale